Library/Homebrew/extend/refinements.rb

```diff
@@ -0,0 +1,20 @@
+# Disable refinement warning.
+begin
+  old_verbose = $VERBOSE
+  $VERBOSE = nil
+  using Module.new
+ensure
+  $VERBOSE = old_verbose
+end
+
+module Refinements
+  refine Module do
+    def attr_predicate(*attrs)
```

Comment:
Well, first I added this to the `extend/module` monkey-patch. Then I thought, let's not add another monkey-patch, let's try refinements! Something as primitive as using a simple Module didn't even cross my mind. 😄